graph.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
Looks awesome! Can you add a brief comment describing what it does and how to use it?